app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
```

Comment:
Please add `new` action here in order to add a form for a new comment on a post. Please see the below error.
![new_action](https://user-images.githubusercontent.com/35504149/155699638-03ec3313-f8fb-400f-a656-90ab8aec6724.png)

Reply:
resolved